jenkins/post-build-status-update.jenkinsfile

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env groovy
+
```

Comment:
Might be nice to get into the habit of a comment here explaining what the pipeline is for, something like:

```groovy
/*
 * Sends the status of a node-test-pull-request build to the Github Bot, which
 * then updates the PR's CI status on github.com.
 */
```